VERSION

```diff
@@ -1 +1 @@
-v1.5.0
+v1.5.1
```

Comment:
this is needed for before release, but helm-version.html is needed after, so should be split up in 2 PRs imo